src/test/java/seedu/address/logic/parser/getparsers/GetAppointmentByDateCommandParserTest.java

```diff
@@ -22,6 +27,28 @@ public void parse_emptyArg_throwsParseException() {
                 String.format(MESSAGE_INVALID_COMMAND_FORMAT, GetAppointmentByDateCommand.MESSAGE_USAGE));
     }
 
+    @Test
+    public void parse_garbledArg_returnsGetAppointmentByDateCommand() {
```

Comment:
think the last part is supposed to be `exceptionThrown` not `returnsGetAppointmentByDateCommand`